lib/config/rpcProviderProdConfig.json

```diff
@@ -37,9 +37,10 @@
   },
   {
     "chainId": 42161,
-    "useMultiProviderProb": 0.1,
-    "providerInitialWeights": [1, 0, 0, 0],
-    "providerUrls": ["INFURA_42161", "QUICKNODE_42161", "NIRVANA_42161", "ALCHEMY_42161"]
+    "useMultiProviderProb": 1,
+    "providerInitialWeights": [1, -1, -1, -1],
+    "providerUrls": ["NIRVANA_42161", "ALCHEMY_42161", "INFURA_42161", "QUICKNODE_42161"],
```

Comment:
you are about to send 100% of traffic to nirvana for 42161?

Reply:
Sorry this is for debug. Reverted already.